### ISSUE TITLE
Any new weights?

### ISSUE THREAD

**Issue by pcengine** [pangafu/LeelaMasterWeight#10]
Hello, are you still training any new weights? It's been more than a month since you uploaded some leelamaster weights. I wonder if this project is still alive or not?

**Comment by anonymousAwesome** [comment]
I would also like to know this.  I really like learning from Leela Master GX, but I'd like one based on stronger, more recent weights.

**Comment by pangafu** [comment]
Yes, I will restart the process next week, and will make 40% human leela master weight

**Comment by pangafu** [comment]
Update... add GX3A, GX3B and GX41 leela master weight
Please enjoy~

**Comment by pcengine** [comment]
@pangafu  
Great! I've tested them all, and it seems GX3B and it works great for handicap games.
